### DISCUSSION TITLE
Accessing Management API with System user credentials

### DISCUSSION THREAD

**Discussion by jasper95** [zitadel/zitadel#4757] [Q&A] [answered]
Im able to create system user token. With this token I created an instance with an owner user. Is there a way that I can create a management api token with the owner user credentials I created? I know this can be done with console but Im finding a way do it via rest api only. Thanks!

**Comment by hifabienne** [comment, marked as answer]
Hei @jasper95 
You have multiple possibilities to access the management apis.
It depends if you are having a human or a service user.

With a human user you need to do a login flow. If you are using an API client like Insomnia or Postman you can configure the flow and login before sending your request. (I use Insomnia and this works perfectly fine) If you need more information how to configure just let me know.

If you are using a service user using Personal Access Token would be an easy way. You can find the documentation here: https://docs.zitadel.com/docs/guides/integrate/pat

**Reply by jasper95** [reply]
Hi @hifabienne  thanks for your response, I appreciate it.

I got what you're trying to say. My use case is I want to generate access management api programatically without going into the console ui given I have system user access token.

Here's the following scenario.

1. I have system user access token
2. I use system user token to create an instance with an owner user
3. Now I have an instance, I need to generate management api token so I can create users,orgs,projects, etc without going to the console ui. How do I do it in this step?

**Reply by hifabienne** [reply]
Ok I got it. 
So at the moment it is only possible to create an instance with an human user as first administrator.
But we already have an open PR that is in review right now, that allows you to create an instance with a service account as first user, so you don't need human interaction afer:
https://github.com/zitadel/zitadel/issues/4417
